README.md

```diff
@@ -1 +1,220 @@
-# docker-plextraktsync
\ No newline at end of file
+<!-- DO NOT EDIT THIS FILE MANUALLY  -->
+<!-- Please read the https://github.com/linuxserver/docker-plextraktsync/blob/main/.github/CONTRIBUTING.md -->
```

Comment:
There are a few `linuxserver`s that need to be `linuxserver-labs` that have been missed in previous iterations of the readme